src/ui/common/src/reducers/pages/Workflow.ts

```diff
@@ -0,0 +1,61 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export type PerDagPageState = {
+  // TODO: Add selected nodes, etc. here
```

Comment:
Should we reference a linear task for this?

Reply:
we will refine this in follow up PRs. This TODO will be resolved once the whole stack is merged